integration-tests/bats/log.bats

```diff
@@ -632,3 +632,15 @@ teardown() {
     run dolt commit --allow-empty -m "% should be escaped"
     [[ "$output" =~ "% should be escaped" ]] || false
 }
+
+@test "log: identify HEAD" {
+    dolt commit --allow-empty -m "commit 1"
+    dolt tag commit1
+    dolt commit --allow-empty -m "commit 2"
+    dolt tag commit2
+    run dolt log commit1
+    [[ !("$output" =~ "HEAD") ]] || false
+    dolt log commit2 | cat
```

Comment:
Seems like you don't need this line and the following line that calls the same command with `run`

```suggestion
```

Reply:
Whoops. That was leftover from when I was debugging the test. Removed.